test/crud_app_web/views/error_view_test.exs

```diff
@@ -4,11 +4,11 @@ defmodule CrudAppWeb.ErrorViewTest do
   # Bring render/3 and render_to_string/3 for testing custom views
   import Phoenix.View
 
-  test "renders 404.html" do
-    assert render_to_string(CrudAppWeb.ErrorView, "404.html", []) == "Not Found"
+  test "renders 404.json" do
+    assert render(CrudAppWeb.ErrorView, "404.json", []) == %{errors: %{detail: "Endpoint Not Found"}}
   end
 
-  test "renders 500.html" do
-    assert render_to_string(CrudAppWeb.ErrorView, "500.html", []) == "Internal Server Error"
+  test "renders 500.json" do
+    assert render(CrudAppWeb.ErrorView, "500.json", []) == %{errors: %{detail: "Internal Server Error"}}
```

Comment:
A minor point, but if you store the error messages as constants then assert on the constants, it's easier to maintain the tests since you won't have to update the copy in two places.

Reply:
It seems like having a constants defined separately is not a common pattern in Elixir. I'm going to do more digging into this and figure out what the best way is! For now, I defined constants for use within the module, which is not very helpful yet, but a start! Will have this done by the next PR. https://github.com/hyoyou/crud-app-elixir/pull/18/commits/7069aca6cb354d7bde57c367025fcec1c4b224fa